src/store/base.rs

```diff
@@ -0,0 +1,200 @@
+use std::fmt::Debug;
+
+use crate::model::{PacBuild, Repository};
+use crate::store::errors::StoreError;
+use crate::store::filters::{InstallState, Kind};
+
+pub type StoreResult<T> = Result<T, StoreError>;
+
+pub trait Base: Debug {
+    /// Removes `PACBUILD` by name that belongs to the given repository.
+    ///
+    /// # Errors
+    /// * `StoreError::RepositoryNotFound`
```

Comment:
```suggestion
    /// * [StoreError::RepositoryNotFound]
```

Use square brackets to link to stuff, replace everywhere.

Reply:
yeah, tell that to clippy